### PR TITLE
Fixes #17034: Add better documentation on how to access technique parameter within an ncf technique

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/17034

### CODE REVIEW

File: src/reference/modules/reference/pages/variables.adoc [hunk 1]
```diff
@@ -541,6 +541,19 @@ site-specific elements...
 
 NOTE: These variables are not available on Windows nodes, but only on with the classic Linux/AIX agent and within the `Technique Editor`.
 
+==== Technique parameters
```
Comment (amousset) [inline src/reference/modules/reference/pages/variables.adoc:544]: Could you also add that this prefixed syntax is required when the parameter is used outside of the technique: in a template or for generic methods taking a variable name (and not a value)

Comment (Fdall) [reply src/reference/modules/reference/pages/variables.adoc:544]: sure

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

**Pull request merged**